setup.py

```diff
@@ -10,18 +10,19 @@
 
 
 requirements = [
-    # TODO(sjperkins)
-    # Depend on actual releases once this branch is merged
     'codex-africanus[dask] >= 0.3.2',
-
-    # TODO(sjperkins)
-    # Depend on actual releases
-    'dask-ms >= 0.2.6',
-
+    # breakage dask-ms 0.2.12
+    # issue 254
+    'dask-ms >= 0.2.6, <= 0.2.11',
+    'progress >= 1.6',
+    'fsspec',
+    'numpy <= 1.19.5',
     'loguru']
 
 extras_require = {
-    'testing': ['pytest', 'pytest-flake8']
+    'testing': ['pytest',
+                'pytest-flake8',
+                'flake8 >= 4.0.0, <5.0.0']
```

Comment:
same as in tricolour. No bandwidth right now to work around API breakage introduced by 5.0.0. It looks like flake8 integration into pytest is discouraged so perhaps we should just not run it the way we do right now

Reply:
Agreed. The way forward here is to use pre-commit hooks, but this can be done in a later release and PR.